### PR TITLE
Feature/models

### PR DESCRIPTION
**Pull request by vijayanands**
Added Model Object Classes and a list arrays for ingredients, their units and cooking techniques

### CODE REVIEW

**Comment by vijayanands** [comment]
I will try to rebase with master and recreate the pull request

**Comment by vijayanands** [comment]
Having some trouble merging my local branch so reopening this PR, I think the conflict can be easily resolved. So leaving it as is

File: RecipeApp/Models/Cookbook.swift [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+//  Cookbook.swift
+//  RecipeApp
+//
+//  Created by Vijayanand on 10/12/17.
+//  Copyright © 2017 The Recipe App. All rights reserved.
+//
+
+import UIKit
+
+class Cookbook: NSObject {
```
Comment (ccavazos) [inline RecipeApp/Models/Cookbook.swift]: Shouldn't this (and all the ones below) be a class of `PFObject` or the base object from Parse?

Comment (vijayanands) [reply RecipeApp/Models/Cookbook.swift]: @ccavazos - I think you have a good point. I missed that. I think they should be.. As far as I know we have to make them an object of type PFObject to be able to store them using Parse.. I will have to fix it.. Will check the same tonight and will do the fix if needed.

**Pull request merged**